branch.properties

```diff
@@ -1,30 +0,0 @@
-category=Developer
```

Comment:
Deleting this file will cause the generation job to fail. (Why do you think I left it in place this long? And why do you think I keep asking for maintenance on the generator?)